svelte.config.js

```diff
@@ -14,7 +14,6 @@ export default {
       $styles: resolve('./src/styles'),
       $atoms: resolve('./src/lib/atoms'),
       $typography: resolve('./src/lib/typography'),
-      $colors: resolve('./src/lib/tailwindColors'),
```

Comment:
Do we still need JS access to the colors, even if we don't want it to be connected with tailwind at all?